### ISSUE TITLE
Plunk

### ISSUE THREAD

**Issue by rameshbas** [valor-software/ngx-bootstrap#384]
I tried your install instructions. None of them worked. Do you have a plunk project?
All Angular2 projects works for me.

**Comment by valorkin** [comment]
like this https://github.com/valor-software/angular2-quickstart ?

**Comment by CarlCui-AI** [comment]
@rameshbas I've made this plunker for reporting an issue, but I guess it might be helpful for you. 
http://plnkr.co/edit/xT6CepFSh6uwiC3TpfRw?p=info

**Comment by rameshbas** [comment]
Thanks. I can try your samples now.

```
  From: Carl Cui <notifications@github.com>
```

 To: valor-software/ng2-bootstrap ng2-bootstrap@noreply.github.com 
Cc: rameshbas rameshbas@yahoo.com
 Sent: Monday, April 11, 2016 1:15 PM
 Subject: Re: [valor-software/ng2-bootstrap] Plunk (#384)

@rameshbas I've made this plunker for reporting an issue, but I guess it might be helpful for you. 
http://plnkr.co/edit/xT6CepFSh6uwiC3TpfRw?p=info—

**Comment by rameshbas** [comment]
I tried to use your datepicker or tabs. Did not work  Please post a plunker showing a demo of all your samples if you have time. It will help the usage of your library.

```
  From: Carl Cui <notifications@github.com>
```

 To: valor-software/ng2-bootstrap ng2-bootstrap@noreply.github.com 
Cc: rameshbas rameshbas@yahoo.com
 Sent: Monday, April 11, 2016 1:15 PM
 Subject: Re: [valor-software/ng2-bootstrap] Plunk (#384)

@rameshbas I've made this plunker for reporting an issue, but I guess it might be helpful for you. 
http://plnkr.co/edit/xT6CepFSh6uwiC3TpfRw?p=info—

**Comment by CarlCui-AI** [comment]
@rameshbas http://valor-software.com/ng2-bootstrap/ ? It's got all the demos. Maybe you can post your plunk so that we can see what went wrong.

**Comment by rameshbas** [comment]
I made another attempt. It is working now.  Alert and tabs Thanks for your help.

```
  From: Carl Cui <notifications@github.com>
```

 To: valor-software/ng2-bootstrap ng2-bootstrap@noreply.github.com 
Cc: rameshbas rameshbas@yahoo.com
 Sent: Monday, April 11, 2016 1:15 PM
 Subject: Re: [valor-software/ng2-bootstrap] Plunk (#384)

@rameshbas I've made this plunker for reporting an issue, but I guess it might be helpful for you. 
http://plnkr.co/edit/xT6CepFSh6uwiC3TpfRw?p=info—
